ansibullbot/triagers/plugins/performance.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+
+def get_performance_facts(issuewrapper, meta):
+    iw = issuewrapper
+
+    pfacts = {
+        'is_performance': False
+    }
+
+    body = iw.body.lower()
+    title = iw.title.lower()
+
+    # TODO search in comments too?
+    for data in (body, title):
```

Comment:
Shorter:
```python
pfacts['is_performance'] = any('performance' in d for d in (body, title))
```

Can also be combined with `pfacts` declaration on line 6

Reply:
Also, `any` adds a bit of semantics to the code, which improves readability. I would likely to understand it much faster when I don't have to scan through 4 lines of loop.

Reply:
I was going to say that it decreases readability after reading the first comment :) Matter of taste I guess, I'll let @jctanner decide.